lib/modules/platform/gerrit/index.md

```diff
@@ -0,0 +1,72 @@
+# Gerrit
+
```

Comment:
Can you add a section where you explain what versions of Gerrit we support?

The Gerrit community only supports the last 3 releases.

> ## Supported Versions
> 
> The Gerrit open-source community actively supports the last 3 releases on a best effort basis. Older releases are not actively maintained but may still receive important fixes (e.g. security fixes), but there is no guarantee for this. Which fixes are backported to these old releases is decided on a case by case basis.
> 
> End of life for old release happens implicitly when a new Gerrit version is released, and is announced via the [project news](https://www.gerritcodereview.com/news.html) and on the mailing list.
> 
> The following table shows the current level of support for Gerrit releases:
> 
> Version | Support Status | Notes
> -- | -- | --
> 3.8 | Active |  
> 3.7 | Active |  
> 3.6 | Active |  
> 3.5 | EOL | EOL since May 19, 2023
> 3.4 | EOL | EOL since Nov 9, 2022
> 3.3 | EOL | EOL since May 24, 2022
> 3.2 | EOL | EOL since Dec 7, 2021
> 3.1 | EOL | EOL since May 19, 2021
> 3.0 | EOL | EOL since December 1st, 2020
> 2.16 | EOL with Support | EOL since June 1st, 2020
> 2.15 | EOL | EOL since November 15th, 2019
> 2.14 | EOL | EOL since May 31st, 2019
> 2.13 | EOL |  
> pre 2.13 | EOL |